src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -424,7 +424,6 @@ public virtual bool CanExecuteText(string text)
 
         public Task<ExecutionResult> ResetAsync(bool initialize = true)
         {
-            GetInteractiveWindow().AddLogicalInput(_interactiveCommands.CommandPrefix + "reset");
```

Comment:
What does ResetAsync "do" now? (seems like it would just say "Resettting..." and do nothing...)